### ISSUE TITLE
Server endpoint for current level (improved level loading)

### ISSUE THREAD

**Issue by Zemanzo** [Zemanzo/web-marbles#184]
**Description**
Currently, the client waits for a websocket connection to establish, which returns the current level ID so it can be downloaded. This can be improved by creating a server endpoint that always serves the current level.

**Minimum Acceptance Criteria**
- [ ] Have an endpoint on the server that serves the currently active level.
- [ ] Have the client fetch the level right away, without waiting on websockets.

**Comment by Zemanzo** [comment]
This is only an optimization, thus low priority for now.
